### DISCUSSION TITLE
Export/Import a Cookbook as a file

### DISCUSSION THREAD

**Discussion by andrew-codechimp** [mealie-recipes/mealie#3931] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I would like to share certain recipes with friends who have their own local mealie instance (and we don't want to expose them publicly).

### Please provide a concise description of the feature that would resolve your issue.

Add an ability to download a cookbook as a file (json?) for sending to someone else who can then use an import function to add the recipes and create the cookbook.

### Please consider and list out some caveats or tradeoffs made in your design decision

Would have to have previews/ability to edit on the import, particularly if you have an existing cookbook with that name.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by DavidN78** [comment]
Are there a lot of recipes? I don't know enough about JSON, but could it work if you go to edit each recipe, click the JSON button then copy/paste the code...?
(I landed here because I'm wondering if I can convert my hundreds of Word-file recipes to a form I can import. All the Word files use the same template - I just need to 'wrap' the text with the JSON syntax. Well I say 'just'...!)
